nucliadb/nucliadb/ingest/orm/entities.py

```diff
@@ -65,22 +65,30 @@ def __init__(
         self.txn = txn
         self.kbid = self.kb.kbid
 
+    async def create_entities_group(self, group: str, entities: EntitiesGroup):
+        # XXX: It's useful enough to check existence to return
+        # if await self.entities_group_exists(group):
+        #     raise AlreadyExists(f"Entities group {group} already exists")
+
```

Comment:
If we decide to not do this check, I would remove the comment and all the related logic (try catch, etc). We can always add it back if we change our minds

Reply:
I'll uncomment it and enable the feature